docs/sidebars.js

```diff
@@ -62,6 +62,11 @@ const sidebars = {
           id: "features/scheduling",
           label: "Scheduling",
         },
+        {
+          type: "doc",
+          id: "features/stale-detection",
+          label: "Stale Detection",
```

Comment:
With "Stale Detection" I don't know what is going stale or what is being detected. I know space in the left sidebar is an issue, but I do see other 2-line subheadings so I think it's OK to lengthen this to "Stale Feature Flag Detection", which is much more descriptive and better for searching.

Reply:
Just want to point out that this nav item in the left sidebar is nested under 'Features', so there is some context there. The title of the actual page is 'stale feature flag detection' so searches should still hit it. Does that clarification make it ok or would you still prefer it more explicit?

Reply:
<img width="342" alt="image" src="https://github.com/growthbook/growthbook/assets/2374625/d78e9b84-ac93-43b7-928c-5a0f5e8e8e43">

Reply:
How about "Stale Feature Detection"? I think we need "Feature" in the subheading but we can drop "flag". I'm seeing "Feature" in some of the other subheadings so "Stale Feature Detection" would fit right in.